src/database/DataTypes/Period.js

```diff
@@ -48,7 +49,9 @@ export class Period extends Realm.Object {
   }
 
   toString() {
-    return `${this.startDate.toLocaleDateString()} - ${this.endDate.toLocaleDateString()}`;
+    const startDate = moment(this.startDate).format('DD/MM/YY');
```

Comment:
I went with a 
```
const DATE_FORMAT = 'DD/MM/YY';

export const formatDate = date => moment(date).format(DATE_FORMAT);
```

in `localization/utilities` and used that. Possibly overkill, but we may need date formatting elsewhere and may need to have a localisation friendly format?

Reply:
Fair enough. There's a few other hardcoded formatting strings so I'll make a new issue for this 👍 .